doc/specs/vulkan/chapters/resources.txt

```diff
@@ -3914,10 +3914,11 @@ ifndef::VK_VERSION_1_1,VK_KHR_dedicated_allocation[]
     dedicated for a specific buffer or image
 endif::VK_VERSION_1_1,VK_KHR_dedicated_allocation[]
 endif::VK_NV_dedicated_allocation[]
-ifndef::VK_VERSION_1_1,VK_KHR_device_group[]
-  * [[VUID-VkBindImageMemoryInfo-memory-01625]]
-    pname:memory must: be a valid dname:VkDeviceMemory handle
-endif::VK_VERSION_1_1,VK_KHR_device_group[]
+ifndef::VK_VERSION_1_1+VK_KHR_swapchain[]
+ifndef::VK_KHR_device_group+VK_KHR_swapchain[]
+  * pname:memory must: be a valid dname:VkDeviceMemory handle
```

Comment:
I'm not entirely sure dropping the VUID is appropriate here.  We're not creating a new VU statement or even modifying an existing VU statement per-say.  We're just fixing the cases in which an existing VU is exposed.

Reply:
I wouldn't know.

The styleguide is bit vague on this case.
And I am not sure how the scripts are handling it.
Change would break layers, but at the same time alert that a change in code is necessary to catch this.

Your call; change, or keep?

Reply:
You can keep the VUID, we will be able to detect it shortly in the layers